src/components/DivideSubmissions.vue

```diff
@@ -179,8 +188,8 @@ export default {
         },
 
         totalWeight() {
-            const graderWeight = this.graders.reduce(
-                (tot, grader) => tot + (grader.weight || 0),
+            const graderWeight = Object.values(this.graderWeights).reduce(
+                (acc, w) => acc + (w || 0),
                 0,
             );
             return Math.max(graderWeight, 1);
```

Comment:
This is wrong when some grader weights are set but they sum to a value < 1.

```suggestion
            return graderWeight || 1;
```